src/pl/plpython/expected/plpython3_path_guc.out

```diff
@@ -0,0 +1,33 @@
+--
+-- Tests for functions that return set PYTHONPATH guc only for gpdb6 plpython3
+--
+LOAD 'plpython3';
+set plpython3.python_path='/foo';
```

Comment:
you had `show plpython3.python_path` before, why did you remove it? I think it is reasonable to have it since it ensures the default value is empty.

Reply:
because...it may set in the last time in our `ENV`

Reply:
if it is a new `ENV` that is ok